cli/src/commands/i18n.js

```diff
@@ -1,32 +1,25 @@
 const { namespace, reporter } = require('@dhis2/cli-helpers-engine')
 const i18n = require('../lib/i18n')
+const makePaths = require('../lib/paths')
 
 const generate = {
     description:
         'Generate JSON files compatible with i18next from po/pot files',
     builder: {
-        path: {
-            alias: 'p',
-            description:
-                'directory path to find .po/.pot files and convert to JSON',
-            default: './i18n/',
-        },
-        output: {
-            alias: 'o',
-            description: 'Output directory to place converted JSON files.',
-            default: './src/locales/',
-        },
```

Comment:
This makes this a breaking change, unfortunately - we need to keep the `path` and `output` options for backwards compatibility but fall back to `paths.i18nStrings` and `paths.i18nLocales` below